VERSION

```diff
@@ -1 +1 @@
-1.18.0
+2.0.0
```

Comment:
I suspect part of Stripe's automatic hosting of tags on our servers. @ebroder is this no longer in use?